plugins/defaultplugins/aclplugin/vppcalls/interfaces_vppcalls.go

```diff
@@ -14,144 +14,221 @@
 
 package vppcalls
 
-// Note: acl_interface_add_del call is deprecated, replace by acl_interface_set_acl_list
-
 import (
 	"fmt"
 	"time"
 
-	"git.fd.io/govpp.git/api"
+	govppapi "git.fd.io/govpp.git/api"
 	"github.com/ligato/cn-infra/logging"
 	"github.com/ligato/cn-infra/logging/measure"
 	acl_api "github.com/ligato/vpp-agent/plugins/defaultplugins/aclplugin/bin_api/acl"
 	"github.com/ligato/vpp-agent/plugins/defaultplugins/ifplugin/ifaceidx"
 )
 
-// SetACLToInterfacesAsIngress sets ACL to all provided interfaces as ingress.
-func SetACLToInterfacesAsIngress(aclIndex uint32, interfaces []string, swIfIndexes ifaceidx.SwIfIndex, log logging.Logger,
-	vppChannel *api.Channel, stopwatch *measure.Stopwatch) error {
-	var timeLog measure.StopWatchEntry
-	if stopwatch != nil {
-		timeLog = measure.GetTimeLog(acl_api.ACLInterfaceSetACLList{}, stopwatch)
+// ACLInterfaceLogicalReq groups multiple fields to not enumerate all of them in one function call
+type ACLInterfaceLogicalReq struct {
+	aclIndex   uint32
+	interfaces []string
+	ingress    bool
+}
+
+// ACLInterfacesVppCalls aggregates vpp calls related to the IP ACL interfaces
+type ACLInterfacesVppCalls struct {
+	vppChan         *govppapi.Channel
+	swIfIndexes     ifaceidx.SwIfIndex
+	dumpIfaces      measure.StopWatchEntry
+	ifaceSetACLList measure.StopWatchEntry
```

Comment:
Variables `dumpIfaces` and `ifaceSetACLList` should include *stopwatch* in the name so it's clear when it's later used.